eliot/_validation.py

```diff
@@ -156,16 +156,19 @@ def validate(value):
 
 
 
-def fields(**keys):
+def fields(*fields, **keys):
     """
     Factory for for L{MessageType} and L{ActionType} field definitions.
 
+    @param *fields: A L{tuple} of L{Field} instances.
+
     @param **keys: A L{dict} mapping key names to the expected type of the
         field's values.
 
     @return: A L{list} of L{Field} instances.
     """
-    return [Field.forTypes(key, [value], "") for key, value in keys.items()]
+    return list(fields) + [
+        Field.forTypes(key, [value], "") for key, value in keys.items()]
```

Comment:
Might be worth validating that there's duplicate of keys... although possibly the MessageType/ActionType constructors already do that? In which case it's not necessary.